### PR TITLE
SPI Host V1 sign-off

### PR DESCRIPTION
**Pull request by rasmus-madsen**
This PR has two commits
1 to address the few documentation comments made in the review
2 A commit that fixes some compile issues seen on VCS only

This sets the DV status to V1

### CODE REVIEW

**Review by senelson7 — APPROVED** [review]
LGTM

File: hw/ip/spi_host/data/spi_host_testplan.hjson [hunk 1]
```diff
@@ -325,12 +340,39 @@
...
             '''
-    }   
+    }
+    {
+      name: segment_speed_cg
+      desc: '''
+            - Check that we test with transactions that uses different speeds
+              i.e first segment in standard mode, followed by segments in dual or quad mode
+            '''
+    }
```
Comment (weicaiyang) [inline hw/ip/spi_host/data/spi_host_testplan.hjson:376]: as discussed, we want to have a covergroup for passthrough. not sure if you have added it somewhere?

**Review by weicaiyang — APPROVED** [review]
LGTM with a question.

perhaps also change the PR title to `SPI Host V1 sign-off`

File: hw/ip/spi_host/data/spi_host_testplan.hjson (hunk 1 above)
Comment (rasmus-madsen) [reply hw/ip/spi_host/data/spi_host_testplan.hjson:376]: @wecai good catch thank you I didn't push my last update. it is there now.
I will merge once the CI passes

**Pull request merged**